src/OmnichannelChatSDK.ts

```diff
@@ -1021,8 +1024,8 @@ class OmnichannelChatSDK {
         let sessionId = this.sessionId;
 
         if (optionalParams.liveChatContext) {
-            requestId = optionalParams.liveChatContext.requestId;
-            chatToken = optionalParams.liveChatContext.chatToken;
+            requestId = optionalParams.liveChatContext.requestId || this.requestId;
```

Comment:
Why should `this.requestId` and `this.chatToken` be passed?

ChatSDK should not passed any of these because this should be something a `costumer` explicitly passes. It's the "chat" a customer decides to connect back to. We cannot alter it even if it's invalid. Otherwise we could potentially alter the intended behavior of the customer.

We can update the `if` statement to ensure `optionalParams.liveChatContext.requestId` and `optionalParams.liveChatContext.chatToken` are present to validate it's a valid `liveChatContext` format